cmd/nvidia-mig-manager/main.go

```diff
@@ -70,6 +71,8 @@ var (
 	driverRootCtrPath string
 	devRoot           string
 	devRootCtrPath    string
+	driverLibraryPath string
+	nvidiaSMIPath     string
```

Comment:
I understand that we've been using globals for the other values here, but let's rather return these from the function where we extract them.

Reply:
Updated.